public/service_portal/v0.1.0/openapi.json

```diff
@@ -967,6 +967,36 @@
                         "readOnly": true,
                         "title": "State",
                         "example": "The current state of a portfolio item"
+                    },
+                    "display_name": {
+                        "type": "string",
+                        "readOnly": true,
+                        "title": "Display Name",
+                        "example": "The name of the portfolio item to be displayed in the UI"
+                    },
+                    "long_description": {
+                        "type": "string",
+                        "readOnly": true,
+                        "title": "Long Description",
+                        "example": "The longer description of a portfolio item"
+                    },
+                    "provider_display_name": {
```

Comment:
The topological API is naming this attribute just `provider`. Its no problem form me to use either in Ui but i think it might be a good idea to keep the naming consistent.

Reply:
@gmcculloug do you have thoughts on this?

Reply:
I was assuming this was the same as the `distributor` property.

![image](https://user-images.githubusercontent.com/2591569/52961659-74763f80-3369-11e9-8ebc-1293a0b617b7.png)

@Hyperkid123 Do you have an example you can share?

Reply:
@gmcculloug oh sorry i ment to write `distributor` instead of ~~`provider`~~. I am thinking the same, it should be the same.I am just curious why there its name differently. On the UI side we can use one component to render service offerings and portfolio items. So i can do something like:
```
distributor = data.distributor || data.provider_display_name
```

At the end of the day it does not matter that much but i would love to keep the code clean as much as possible.

Reply:
Updated `provider_display_name` to `distributor` on our end.